tests/EmailAddress.test.ts

```diff
@@ -76,7 +76,7 @@ runStandardTestSuite(S.EmailAddress, _ => ({
   eqTests: [],
   jsonSchema: JS.string({
     pattern:
-      '^(([A-Za-z0-9!#$%&\'*+\\x2d/=?\\x5e_`{|}~]+(\\.[A-Za-z0-9!#$%&\'*+\\x2d/=?\\x5e_`{|}~]+)*|"[^"\\x00-\\x1f]+")@(\\[\\d{1,3}\\.\\d{1,3}\\.\\d{1,3}\\.\\d{1,3}\\]|([A-Za-z0-9\\x2d]{0,63}\\.)+[A-Za-z]{2,}))$',
+      '^(([A-Za-z0-9!#$%&\'*+\\x2d\\x2f=?\\x5e_`{|}~]+(\\.[A-Za-z0-9!#$%&\'*+\\x2d\\x2f=?\\x5e_`{|}~]+)*|"[^"\\x00-\\x1f]+")@(\\[\\d{1,3}\\.\\d{1,3}\\.\\d{1,3}\\.\\d{1,3}\\]|([A-Za-z0-9\\x2d]{0,63}\\.)+[A-Za-z]{2,}))$',
```

Comment:
This is a fun, why do you think this changed?

Reply:
It changed in [1.3.2](https://github.com/skeate/kuvio/compare/v1.3.1...v1.3.2) -- the lockfile still had 1.3.1 before this